week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -15,7 +26,14 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
+function validate(num) {
+  let nums = num;
```

Comment:
Do you think this line is necessary? Could we go without it?

Reply:
Yes, I don't need another variable